test/zmumu/tp_from_aod_MC.py

```diff
@@ -77,7 +77,8 @@
 process.muonMatchHLTL3.maxDeltaR = 0.1
 from MuonAnalysis.MuonAssociators.patMuonsWithTrigger_cff import *
 changeRecoMuonInput(process, "mergedMuons")
-#useExtendedL1Match(process) #MM no idea what the sequence did, not available since git migration
+#useL1Stage2Candidates(process)
+#appendL1MatchingAlgo(process)
```

Comment:
Hi, as I'm going to be away for a few days (disappearing ~ now), and I think you are in a hurry for setting up a new version of TnP production for data, I would suggest I look for a proper sample (with reHLT) and I setup everything properly in a future PR, ok?